app/consumer.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from structlog import wrap_logger
+from async_consumer import AsyncConsumer
+import settings
+from response_processor import ResponseProcessor
+
+logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
+
+logger = logging.getLogger(__name__)
+
+logger = wrap_logger(logger)
+
+
+class Consumer(AsyncConsumer):
+    def on_message(self, unused_channel, basic_deliver, properties, body):
+
+        logger.info('Received message', delivery_tag=basic_deliver.delivery_tag, app_id=properties.app_id, body=body)
+
+        processor = ResponseProcessor(logger)
+        proccessed_ok = processor.process(body.decode("utf-8"))
```

Comment:
I think this should be wrapped in a try block - currently if an exception is raised anywhere in the process block that isn't caught sdx-downstream just dies.